src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -176,8 +216,35 @@ export const ChannelView: React.FC = () => {
           </StyledButtonContainer>
         </TitleSection>
         <TabsContainer>
-          <Tabs initialIndex={0} tabs={mappedTabs} onSelectTab={handleSetCurrentTab} />
+          <StyledTabs
+            selected={isSearching ? -1 : undefined}
+            initialIndex={0}
+            tabs={mappedTabs}
+            onSelectTab={handleSetCurrentTab}
+          />
           {currentTabName === 'Videos' && (
+            <SearchContainer>
+              <StyledTextField
+                ref={searchInputRef}
+                isOpen={isSearchInputOpen}
+                value={searchQuery}
+                onChange={(e) => setSearchQuery(e.target.value)}
+                onKeyDown={handleKeyPress}
+                placeholder="Search"
+                type="search"
+                helperText={null}
+              />
+              <SearchButton
+                onClick={() => {
+                  setIsSearchingInputOpen(true)
+                  searchInputRef.current?.focus()
+                }}
+                variant="tertiary"
+                icon={<SvgOutlineSearch2 />}
+              ></SearchButton>
```

Comment:
as I mentioned before, use IconButton
```suggestion
              <SearchButton
                onClick={() => {
                  setIsSearchingInputOpen(true)
                  searchInputRef.current?.focus()
                }}
                variant="tertiary"
              >
                <SvgOutlineSearch2 />
              </SearchButton>
```